### DISCUSSION TITLE
Markdown for a Release/tag notes does support in-page anchor links to headings

### DISCUSSION THREAD

**Discussion by connorjclark** [community/community#48311] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Regular markdown files support jump-links to their own headers. Click around the Table of Contents here for an example: https://gist.github.com/connorjclark/b3147f4019bad30fe9df4aa1d55ed884

However, this exact same markdown content as the notes of a release/tag _will not_ result in a jump link. The links are still blue in the TOC, but clicking them does nothing. For reference, I'm referring to the description found on a page such as [this](https://github.com/ArmageddonGames/ZQuestClassic/releases/tag/2.55-alpha-112) (this is a real release, so you won't find the broken links here. I simply link to a gist instead as a workaround)

**Comment by ezzatron** [comment]
I ran into this same issue with my [GitHub action](https://github.com/ghalactic/github-release-from-tag) that automatically produces releases from tags. I ended up adding a feature to add markdown heading anchors to release bodies for you. For example, here's a link to the exact heading in the release announcing the feature:

https://github.com/ghalactic/github-release-from-tag/releases/v5.3.0#markdown-heading-anchors

If you need to be able to do this, and don't want to use my action, it can be done manually with an HTML tag in the heading:

```markdown
# <a id="example-heading"></a>Example heading

A link to an [example heading](#example-heading).
```

It would be much nicer if GitHub just added the anchors automatically though.
